python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -0,0 +1,1845 @@
+            north celestial pole. Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or `Astropy.Angle`
+        rot_par :  `float`, `str` or `Astropy.Angle`
+            Specify rotation with respect to the parallactic angle.
+            Accepts float (deg), sexagesimal string (DD:MM:SS.S or DD MM SS.S)
+            coordinates or `Astropy.Angle`
+        rot_tel : `float`, `str` or `Astropy.Angle`
+            Specify rotation in mount physical coordinates.
+            Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or `Astropy.Angle`
+        slew_timeout : `float`
+            Timeout for the slew command (second).
+
+        """
+
+        object_table = Simbad.query_object(name)
+
+        if object_table is None:
+            raise RuntimeError(f"Could not find {name} in Simbad database.")
+        elif len(object_table) > 1:
+            self.log.warning(f"Found more than one entry for {name}. Using first one.")
+
+        self.log.info(
+            f"Slewing to {name}: {object_table['RA'][0]} {object_table['DEC'][0]}"
+        )
+
+        await self.slew_icrs(
+            ra=object_table["RA"][0],
+            dec=object_table["DEC"][0],
+            rot_sky=rot_sky,
+            rot_par=rot_par,
+            rot_tel=rot_tel,
+            target_name=name,
+            slew_timeout=slew_timeout,
+        )
+
+    async def slew_icrs(
+        self,
+        ra,
+        dec,
+        rot_sky=0.0,
+        rot_par=None,
+        rot_tel=None,
+        target_name="slew_icrs",
+        slew_timeout=240.0,
+        stop_before_slew=True,
+        wait_settle=True,
+    ):
+        """Slew the telescope and start tracking an Ra/Dec target in ICRS
+        coordinate frame.
+
+        Parameters
+        ----------
+        ra : `float`, `str` or `Astropy.Angle`
+            Target RA, either as a float (hour), a sexagesimal string
+            (HH:MM:SS.S or HH MM SS.S) coordinates or `Astropy.Angle`.
+        dec : `float`, `str` or `Astropy.Angle`
+            Target Dec, either as a float (deg), a sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or `Astropy.Angle`.
+        rot_sky : `float`, `str` or `Astropy.Angle`
+            Specify rotation in sky coordinates, clock-wise, relative to the
+            north celestial pole. Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or `Astropy.Angle`
+        rot_par :  `float`, `str` or `Astropy.Angle`
+            Specify rotation with respect to the parallactic angle.
+            Accepts float (deg), sexagesimal string (DD:MM:SS.S or DD MM SS.S)
+            coordinates or `Astropy.Angle`
+        rot_tel : `float`, `str` or `Astropy.Angle`
+            Specify rotation in mount physical coordinates.
+            Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or `Astropy.Angle`
```

Comment:
The idea is not to hold it in position but to allow the user to specify an initial rotator physical position to track a target. The pointing does support keeping the rotator fixed but this is not the intention here...

Reply:
Then please document this *very* thoroughly here. It's not at all what I would expect an angle named "rot_phys" to do, and I'll be plenty of others will be equally confused. I have no idea what a good name of that would be. Users need to know if it will maintain the instrument orientation relative to the sky or the horizon.

Reply:
This also brings up an obvious question: will users want *both* cases? In other words will  users want to be able to specify the initial angle as a physical angle, but have it then hold the instrument with respect to either the sky *or* the horizon? That'd be a 4th rotation angle parameter!

Reply:
I don't think so... In fact, even this use case is getting less and less attention.... eventually, I hope I'll be able to support `rot_sky` and `rot_par` alone.